client/components/chart/index.js

```diff
@@ -222,6 +260,18 @@ Chart.propTypes = {
 	 * A number formatting string, passed to d3Format.
 	 */
 	yFormat: PropTypes.string,
+	/**
+	 * Information about the currently selected interval, and set of allowed intervals for the chart. See `getIntervalsForQuery`.
+	 */
+	intervalData: PropTypes.object,
+	/**
+	 * Interval specification (hourly, daily, weekly etc).
```

Comment:
I suppose we should note that `day` is the default like we do on line 276  for legend layout

Reply:
I think the documentation creation automatically grabs the default value from `defaultProps`